core/crypto/client_tcert_pool.go

```diff
@@ -23,7 +23,9 @@ type tCertPool interface {
 
 	Stop() error
 
-	GetNextTCert() (tCert, error)
+	GetNextTCerts(nCerts int) ([]tCert, error)
```

Comment:
(Good morning) @adecaro. At this step, not wanting to 'rock the boat' while there are 2 more pending PRs that need this (internal) change, the change is merely to return `[]tCerts` instead of the (currently returned) `tCert`.

That said, let me get back to you on that point (what should be returned externally, and what should be returned externally), after a few more refactoring steps.